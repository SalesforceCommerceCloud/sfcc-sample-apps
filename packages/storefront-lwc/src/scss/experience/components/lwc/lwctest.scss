.lwctest {
    color: chocolate;
    background-color: azure;
}
