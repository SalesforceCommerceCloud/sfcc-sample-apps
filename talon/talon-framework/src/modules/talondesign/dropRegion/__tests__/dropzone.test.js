import { createElement } from 'lwc';
import DropRegion from 'talondesign/dropRegion';


describe('talondesign/dropRegion', () => {
    it('renders', async () => {
        const element = createElement('talondesign-drop-region', { is: DropRegion });
        document.body.appendChild(element);
        expect(element).toMatchSnapshot();
    });
});