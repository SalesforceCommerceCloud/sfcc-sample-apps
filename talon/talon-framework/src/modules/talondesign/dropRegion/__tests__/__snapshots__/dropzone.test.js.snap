// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`talondesign/dropRegion renders 1`] = `
<talondesign-drop-region>
  #shadow-root(open)
    <div
      class="dropzone"
    />
</talondesign-drop-region>
`;
